### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by Kuzeikind**
Configured maven profiles. Added tests.

### CODE REVIEW

File: src/test/resources/com/epam/tc/hw1/SumSubCalculatorTestSuite.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<suite name="All Calculator Test Suite">
+
+  <test name="All Calculator Test">
+    <classes>
+      <class name="com.epam.tc.hw1.CalculatorSumTest"/>
+      <class name="com.epam.tc.hw1.CalculatorSubTest"/>
+    </classes>
+  </test>
+  
+</suite>
```
Comment (akulebyakin) [inline src/test/resources/com/epam/tc/hw1/SumSubCalculatorTestSuite.xml:12]: Suite name and test name seem to be incorrect here. The name is "All Calculator Test" but test classes: CalculatorSumTest, CalculatorSubTest

File: src/test/java/com/epam/tc/hw1/CalculatorSubTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            {0.5, 0.5, 0d},
+            {1.234, 1.233, 0.001}
+        };
+    }
+
+    @Test(dataProvider = "subDoubleProvider")
+    public void subDoubleTest(double a, double b, double expected) {
+        assertThat(calculator.sub(a, b)).isEqualTo(expected, Offset.offset(0.00001));
+
+    }
```
Comment (akulebyakin) [inline src/test/java/com/epam/tc/hw1/CalculatorSubTest.java:38]: Empty line can be removed I suppose

**Review by akulebyakin — CHANGES_REQUESTED** [review]
Small changes are required

File: src/test/java/com/epam/tc/hw1/CalculatorSubTest.java (hunk 2 above)
Comment (Kuzeikind) [reply src/test/java/com/epam/tc/hw1/CalculatorSubTest.java]: Done.

File: pom.xml [hunk 3]
```diff
@@ -13,18 +13,64 @@
...
+                <suite.name>MulDivCalculatorTestSuite.xml</suite.name>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>hw1-sum-sub-calculator-test-suite</id>
+            <properties>
+                <suite.name>SumSubCalculatorTestSuite.xml</suite.name>
+            </properties>
+        </profile>
```
[the remark this answers is not in the document]
Comment (Kuzeikind) [reply pom.xml:51]: Done.

**Pull request merged**